modules/auxiliary/scanner/http/owa_login.rb

```diff
@@ -96,7 +96,8 @@ def initialize
 
     register_advanced_options(
       [
-        OptString.new('AD_DOMAIN', [ false, "Optional AD domain to prepend to usernames", ''])
+        OptString.new('AD_DOMAIN', [ false, "Optional AD domain to prepend to usernames", '']),
+        OptFloat.new('BASELINE_AUTH_TIME', [ false, "Baseline HTTP authentication response time for invalid users", 1.0])
```

Comment:
Since this is a totally new option it should be in `CamelCase` like most advanced options:
```suggestion
        OptFloat.new('BaselineAuthTime', [ false, "Baseline HTTP authentication response time for invalid users", 1.0])
```